src/zjs_fs.c

```diff
@@ -326,6 +326,10 @@ static ZJS_DECL_FUNC_ARGS(zjs_fs_unlink, u8_t async)
     }
 
     ret = fs_unlink(path);
+    if (ret != 0) {
+        ERR_PRINT("failed to unlink %s with error [%d]\n", path, ret);
```

Comment:
Let's change this to DBG, that way we save the string ROM unless the user is specifically looking into weird behavior and turns debug on. This is a good policy for very rare errors I think.